tests/tests.cpp

```diff
@@ -12,6 +12,38 @@ using namespace sqlite_orm;
 using std::cout;
 using std::endl;
 
+void testIterateBlob() {
+    struct Test {
+        int64_t id;
+        std::vector<char> key;
+    };
+    
+    auto db = make_storage("",
+                           make_table("Test",
+                                      make_column("key", &Test::key),
+                                      make_column("id", &Test::id, autoincrement(), primary_key())));
+    db.sync_schema(true);
+    
+    std::vector<char> key;
```

Comment:
use below code instead of this

```cpp
std::vector<int> v(255) ; // vector with 255 ints.
std::iota (std::begin(v), std::end(v), 0); // Fill with 0, 1, ..., 254.
```

https://stackoverflow.com/a/17694752/1539100